instrumentation/othttp/common.go

```diff
@@ -28,6 +28,14 @@ const (
 	WriteErrorKey = kv.Key("http.write_error") // if an error occurred while writing a reply, the string of the error (io.EOF is not recorded)
 )
 
+// Server HTTP metrics
+const (
+	RequestCount  = "request_count"  // request count total
+	RequestBytes  = "request_bytes"  // request bytes total
+	ResponseBytes = "response_bytes" // response bytes
```

Comment:
Can these use the same names as the trace attributes above?  Or perhaps the [proposed semantic conventions for HTTP content size](https://github.com/open-telemetry/opentelemetry-specification/pull/641/files) that the above will eventually migrate to.

Reply:
yep, will do

Reply:
done